### ISSUE TITLE
Bumblebee not working

### ISSUE THREAD

**Issue by AsmodeanWoT** [Bumblebee-Project/Bumblebee#815]
So I've been searching for quite a while now how to solve this problem. Nothing seems to work and it's really frustrating. I have an Acer Aspire  E5-575G56Y with an Intel i5-6200U and a NVidia 950M and I'm running Antergos. And I just can't seem to get Bumblebee working. The initial problem was, that I was getting a "GPU driver could not be loaded" error. I tried different things and then it seemed to kinda work, I wasn't getting the error anymore but starting games wouldn't really work and it was very inconsistent, meaning sometimes it would work and sometimes I would still get the "GPU driver not found" error. Now I'm back to getting the error message everytime. Hope someone can help me.

**Comment by AsmodeanWoT** [comment]
`[asmodean@maschine ~]$ optirun --status
Bumblebee status: Ready (3.2.1). X inactive. Discrete video card is on.

[asmodean@maschine ~]$ optirun pwd
[   55.826609] [ERROR]Cannot access secondary GPU - error: Could not load GPU driver

[   55.826800] [ERROR]Aborting because fallback start is disabled.`

`[asmodean@maschine ~]$ sudo bumblebeed --debug
[sudo] password for asmodean: 
[   36.888436] [DEBUG]bbswitch has been detected.
[   36.888452] [INFO]Switching method 'bbswitch' is available and will be used.
[   36.888455] [DEBUG]Active configuration:
[   36.888457] [DEBUG] bumblebeed config file: /etc/bumblebee/bumblebee.conf
[   36.888459] [DEBUG] X display: :8
[   36.888461] [DEBUG] LD_LIBRARY_PATH: /usr/lib/nvidia:/usr/lib32/nvidia
[   36.888463] [DEBUG] Socket path: /var/run/bumblebee.socket
[   36.888465] [DEBUG] xorg.conf file: /etc/bumblebee/xorg.conf.nvidia
[   36.888467] [DEBUG] xorg.conf.d dir: /etc/bumblebee/xorg.conf.d
[   36.888469] [DEBUG] ModulePath: /usr/lib/nvidia/xorg/,/usr/lib/xorg/modules
[   36.888471] [DEBUG] GID name: bumblebee
[   36.888473] [DEBUG] Power method: auto
[   36.888475] [DEBUG] Stop X on exit: 1
[   36.888477] [DEBUG] Driver: nvidia
[   36.888481] [DEBUG] Driver module: nvidia
[   36.888487] [DEBUG] Card shutdown state: 1
[   36.888557] [DEBUG]Process /sbin/modprobe started, PID 1277.
[   36.888584] [DEBUG]Hiding stderr for execution of /sbin/modprobe
[   36.889437] [DEBUG]SIGCHILD received, but wait failed with No child processes
[   36.889452] [DEBUG]Configuration test passed.
[   36.889613] [INFO]bumblebeed 3.2.1 started
[   37.033916] [INFO]Switching dedicated card OFF [bbswitch]
[   37.035614] [WARN]Unable to disable discrete card.[   37.035633] [INFO]Initialization completed - now handling client requests
[   47.873189] [DEBUG]Accepted new connection
[   48.021186] [DEBUG]Socket closed.
[   55.349538] [DEBUG]Accepted new connection
[   55.497576] [INFO]Loading driver nvidia (module nvidia)
[   55.497806] [DEBUG]Process modprobe started, PID 1289.
modprobe: ERROR: could not insert 'nvidia': No such device
[   55.825818] [DEBUG]Process with PID 1289 returned code 1
[   55.826500] [ERROR]Module nvidia could not be loaded (timeout?)
[   55.826526] [ERROR]Could not load GPU driver
[   55.826796] [DEBUG]Socket closed.
`
This is what I get, when running the Bumblebee debug, while trying to turn off the nvidia gpu right after a reboot. After that it stops working completely until I restart the laptop.

**Comment by edoantonioco** [comment]
Isnt this related to #810?

**Comment by AsmodeanWoT** [comment]
Yeah that seems to have fixed the not being able to turn it off issue, thanks! :)

**Comment by AsmodeanWoT** [comment]
But it still acts weird when i try to run games with it

**Comment by Lekensteyn** [comment]
Some applications show what "GL renderer" they use. If you see something with "Mesa DRI Intel", then it is your iGPU, otherwise it is likely your dGPU.
